src/applications/vaos/services/appointment/index.js

```diff
@@ -128,6 +128,8 @@ export async function getBookedAppointments({
     if (useV2VA || useV2CC) {
       const allAppointments = await getAppointments(startDate, endDate, [
         'booked',
+        'arrived',
```

Comment:
The swagger docs also include a `noshow` status. Would that status be redundant with `cancelled` being included?

Reply:
It looks like we currently try not to display appointments that have the no show statuses (they're in one of our excluded status lists), so I didn't include them.